main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2353,11 +2354,9 @@ public void call() {
     }
 
     public static void editPersonalNote(final Geocache cache, final CacheDetailActivity activity) {
-        if (cache.isOffline()) {
```

Comment:
@rsudev If we are into defensive programming, I would prefer an `assert()` here in this case since this condition *cannot* happen.